ebean-test/testconfig/ebean-cockroach.properties

```diff
@@ -1,4 +1,5 @@
 ebean.test.platform=cockroach
 ebean.test.dbName=unit
 datasource.default=cockroach
+#ebean.test.cockroach.version=v21.2.6
```

Comment:
We need at least v21.2 here, otherwise the test will not pass
Requires https://github.com/ebean-orm/ebean-test-docker/issues/67 first